src/llvm_tools.rs

```diff
@@ -78,6 +79,18 @@ pub fn profraws_to_lcov(
     Ok(results)
 }
 
+fn get_profdata_path() -> Result<PathBuf, String> {
+    let path = Tool::Profdata.path().map_err(|x| x.to_string())?;
+    if !path.exists() {
+        Err(String::from(
+            "We couldn't find llvm-profdata. \
+             Do you have the llvm-tools component installed?"
```

Comment:
Nit: can you also say how to install it?

Reply:
Definitely, added in b2ac731